app/models/package_manager/cocoa_pods.rb

```diff
@@ -50,9 +50,7 @@ def self.mapping(raw_project)
 
     def self.versions(raw_project, _name)
```

Comment:
🎱 Should there be a spec or something to test these lines of code before it hits prod?